src/components/gameCard/gameCard.tsx

```diff
@@ -17,17 +17,23 @@ interface IGameCardProps {
 
 const GameCard = (props: IGameCardProps) => {
   const dispatch = useDispatch();
-  const userName = useSelector((state: RootState) => state.user.userName);
+  const user = useSelector((state: RootState) => state.user);
+
+  const isLogged = user.userName !== undefined;
+  const { isAdmin } = user;
 
   const getCard = () => {
-    if (userName) {
+    if (isLogged) {
       dispatch(addGame(props.gameInfo));
       alert("Got product");
     } else {
       alert("Sign in first!");
     }
   };
 
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  const adminForm = <AdminForm formInfo={formByName("Edit card")} gameInfo={props.gameInfo} onExit={() => {}} />;
+
```

Comment:
empty onExit function?????? remove all useless functions. also in products, gameCard, profile

Reply:
empty functions all over the project removed / replaced with 'undefined'